modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -21,10 +21,11 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. Optional: You can import a devfile, a Dockerfile, or a builder image through your Git repository to further customize your deployment.
-* If your Git repository contains a devfile, a Dockerfile, or a builder image, it is automatically detected and populated on the respective path fields. If a devfile, a Dockerfile, and a builder image are detected in the same repository, the devfile is selected by default.
+. Optional: You can import a `Devfile`, a `Dockerfile`, or a `Builder Image` through your Git repository to further customize your deployment.
+* If your Git repository contains a `Devfile`, a `Dockerfile`, or a `Builder Image`, it is automatically detected and populated on the respective path fields. If a `Devfile`, a `Dockerfile`, or a `Builder Image` are detected in the same repository, the `Devfile` is selected by default.
 * To edit the file import type and select a different strategy, click *Edit import strategy* option.
-* If multiple devfiles, Dockerfiles, or builder images are detected, to import a specific devfile, Dockerfile, or a builder image, specify the respective paths relative to the context directory.
+* If multiple `Devfiles`, a `Dockerfiles`, or a `Builder Images` are detected, to import a specific `Devfile`, a `Dockerfile`, a `Builder Image`, specify the respective paths relative to the context directory.
+* If GitHub App is set up, you can see *Use GitHub App* and *Setup a webhook* options. If GitHub App is not set up, you can see *Setup a webhook* option.
```

Comment:
@shreyasiddhartha this is only shown when the user checks the configure pac radio button. This is a part of the form that is shown when the above checkbox is checked.

Reply:
@divyanshiGupta Sure. I'll add this as a note in the Pipelines section then as it would make more sense there.